code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -631,18 +638,20 @@
 	for (var/mob/living/carbon/human/H in L)
 		step_away(H, src, sweep_range, 2)
 		if(H.stat != DEAD)
-			var/damage = rand(melee_damage_lower,melee_damage_upper) * 1.25
+			var/damage = rand(melee_damage_lower,melee_damage_upper)
 			if(frenzy_aura > 0)
 				damage += (frenzy_aura * 2)
 			var/affecting = H.get_limb(ran_zone(null, 0))
 			if(!affecting) //Still nothing??
 				affecting = H.get_limb("chest") //Gotta have a torso?!
-			var/armor_block = run_armor_check(affecting, "melee")
-			H.apply_damage(damage, BRUTE, affecting, armor_block)
+			var/armor_block = H.run_armor_check(affecting, "melee")
+			H.apply_damage(damage, BRUTE, affecting, armor_block) //Crap base damage after armour...
+			H.apply_damage(damage, HALLOSS) //...But some sweet armour ignoring Halloss
```

Comment:
This might be very strong crowd control, considering it's a guaranteed hit. Let's see how it rolls.

Reply:
12 second cooldown should keep it sane; also considering increasing the plasma cost so it actually matters.